src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,29 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            actor.setId((Long) session.save(actor));
```

Comment:
why are you doing this
you dont have to explicitly set id
![image](https://user-images.githubusercontent.com/81781479/195890206-ef3767a2-d4fa-4c4e-8b28-81c824d76763.png)